--- scripts/kconfig/lxdialog/dialog.h	2023-01-05 09:29:10.000000000 -0600
+++ scripts/kconfig/lxdialog/dialog.h.patched	2025-11-17 14:26:19.970812736 -0600
@@ -28,6 +28,14 @@
 #ifdef __sun__
 #define CURS_MACROS
 #endif
+
+/* Si el script de chequeo no definió CURSES_LOC, 
+ * asumimos ncurses estándar.
+ */
+#ifndef CURSES_LOC
+# define CURSES_LOC <ncurses.h>
+#endif
+
 #include CURSES_LOC
 
 /*
